doc/user-guide/inputs/secondary-perils-inputs.rst

```diff
@@ -0,0 +1,70 @@
+.. _secondary-perils:
+
+Secondary perils
+================
+
+Several methodologies exist for calculating probabilities and displacements from secondary perils, such as landslides
+and liquefaction. We have implemented multiple models in OpenQuake, each requiring different input datasets with global
+coverage. These inputs are incorporated by adjusting the site model. Performing secondary perils calculations requires 
+the adjusted `site_model.csv` in addition to the required files for running event-based or scenario analyses. 
+Futhermore, user is asked to define in `job.ini` file which model should be used to perform the analysis using the
+parameter ``secondary_perils``. For example, if one wants to use the HAZUS methodology, they should type::
+    
+    secondary_perils = HazusLiquefaction
+
+Before demonstrating an example of a site model, we first discuss the implemented functions that are useful for 
+retrieving relevant inputs for sites.
+
+
+Getting raster values at sites
+------------------------------
+
+Digital elevation data and its derivatives are often given as rasters. However, in the case of probabilistic analysis 
+of secondary perils (particularly for risk analysis) the analyist may need to deal with sites that are not distributed 
+according to a raster grid.
+
+Raster values may be extracted at sites using a GIS program to perform a spatial join, but following inconvenient 
+historical precedent, this operation often produces new data files instead of simply appending the raster values to the 
+point data file.
+
+Therefore we have implemented a simple function, `srap <https://github.com/gem/oq-engine/blob/ef33b5e0dfdca7a214dac99d4d7214086023ab39/openquake/sep/utils.py#L22>`_,
```

Comment:
`sample_raster_at_points(...) <https://github.com/gem/oq-engine/blob/engine-3.20/openquake/sep/utils.py#L19>`_

use full function name and link to a specific engine release